psalm.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0"?>
 <psalm
         errorLevel="1"
+        phpVersion="7.4"
```

Comment:
Sorry, wasn't clear - the PHP version should probably be 8.1, which is where psalm is run :)

Reply:
I understood that part but got confused because we are still running `ci` check on the minimum supported PHP version,`PHP 7.4`.

Reply:
Yeah, I'm generally confused by the CI output anyway :shrug: 

Probably need to get back to this at a later point in time, checking it out locally, because those CI failures should really not be there, unless the PHP version changed some details internally too :thinking: